.travis.yml

```diff
@@ -1,17 +1,13 @@
-sudo: false
-addons:
-  apt:
-    packages:
-      - libxml2-dev
-      - libreadline-dev
-      - valgrind
+sudo: true
+dist: trusty
```

Comment:
I am not too familiar with Ubuntu's releases, but why base this on Trusty rather than Xenial ?